API/package.json

```diff
@@ -10,7 +10,12 @@
   "license": "MIT",
   "dependencies": {
     "express": "^4.15.3",
+    "memory-cache": "^0.2.0",
```

Comment:
I really don't feel we need this dependency

Reply:
removed !

Reply:
done